apis/kueue/v1alpha1/multikueue_types.go

```diff
@@ -28,6 +28,9 @@ const (
 type LocationType string
 
 const (
+	// Location is the path on the disk of kueue-controller-manager.
+	PathLocationType LocationType = "Path"
```

Comment:
This `Path` location type is assumed to be the main story that we want to use credentials that existed on the Node. It means that the main use case is mounting credentials by hostPath volume, right?

Reply:
It makes sense. I think that we should mention here that we would suggest using the `SecretLocationType` as much as possible, and the `PathLocationType` is assumed to be used for mounting credentials from nodes via hostPath volumeMount.

However, since this is a non-blocking suggestion, I think that we can work on another PR. I leave on @trasc whether or not we work on it at this PR.

Reply:
`SecretLocationType` is the default value, we could present come pros/cons when we add the documentation.